### ISSUE TITLE
Update for 1.10.2

### ISSUE THREAD

**Issue by EntityReborn** [tterrag1098/WAILAPlugins#70]
We are running FTB HermitPack 1.2.1, which is MC version 1.10.2.
Does this version support it?

**Comment by tterrag1098** [comment]
No, I haven't had time to update this mod, as it would be a bit of work with all the new versions of mods. I'll get to it at some point.

**Comment by EntityReborn** [comment]
Would it be entirely possible to just update the one plugin? Mekanism? Currently with vanilla WAILA, lots of the machine blocks are mislabeled, and I was hoping this would resolve that.
